init-tools.cmd

```diff
@@ -8,6 +8,7 @@ set DOTNET_PATH=%TOOLRUNTIME_DIR%\dotnetcli\
 if [%DOTNET_CMD%]==[] set DOTNET_CMD=%DOTNET_PATH%dotnet.exe
 if [%BUILDTOOLS_SOURCE%]==[] set BUILDTOOLS_SOURCE=https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json
 set /P BUILDTOOLS_VERSION=< "%~dp0BuildToolsVersion.txt"
+if [%ILLINK_SOURCE%]==[] set ILLINK_SOURCE=https://dotnet.myget.org/F/dotnet-core/api/v3/index.json
```

Comment:
Lets try to avoid this. Can we instead put this package on the dotnet-buildtools feed?